### PR TITLE
Harvest spatial geoms

### PR DESCRIPTION
**Pull request by ThibaudDauce**
Related to https://github.com/etalab/data.gouv.fr/issues/763

### CODE REVIEW

**Comment by landryb** [comment]
i've had a very quick look at the code and the XSL transformation, but does this properly analyses xml such as
```
<dct:spatial>
    <ogc:Polygon>
        <geo:asWKT rdf:datatype="http://www.opengis.net/rdf#wktLiteral">
            Polygon((4.44641288 45.54214467, 4.44641288 46.01316963, 4.75655252 46.01316963, 4.75655252 45.54214467, 4.44641288 45.54214467))
        </geo:asWKT>
    </ogc:Polygon>
</dct:spatial>
```
that's what GN4 generates for metadata bboxes...

**Comment by ThibaudDauce** [comment]
> i've had a very quick look at the code and the XSL transformation, but does this properly analyses xml such as
> 
> ```
> <dct:spatial>
>     <ogc:Polygon>
>         <geo:asWKT rdf:datatype="http://www.opengis.net/rdf#wktLiteral">
>             Polygon((4.44641288 45.54214467, 4.44641288 46.01316963, 4.75655252 46.01316963, 4.75655252 45.54214467, 4.44641288 45.54214467))
>         </geo:asWKT>
>     </ogc:Polygon>
> </dct:spatial>
> ```
> 
> that's what GN4 generates for metadata bboxes...

Hi, thanks for the DCAT example, I will try it. Do you have a link to a catalog with these types of spatial coverage so I can do more experiments?

**Comment by landryb** [comment]
> > i've had a very quick look at the code and the XSL transformation, but does this properly analyses xml such as
> > ```
> > <dct:spatial>
> >     <ogc:Polygon>
> >         <geo:asWKT rdf:datatype="http://www.opengis.net/rdf#wktLiteral">
> >             Polygon((4.44641288 45.54214467, 4.44641288 46.01316963, 4.75655252 46.01316963, 4.75655252 45.54214467, 4.44641288 45.54214467))
> >         </geo:asWKT>
> >     </ogc:Polygon>
> > </dct:spatial>
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > that's what GN4 generates for metadata bboxes...
> 
> Hi, thanks for the DCAT example, I will try it. Do you have a link to a catalog with these types of spatial coverage so I can do more experiments?

yes definitely, there's https://demo.georchestra.org/geonetwork/srv/eng/catalog.search#/home which has **a lot** or https://demo.georchestra.org/geonetwork/craig_harvested/eng/csw?SERVICE=CSW&VERSION=2.0.2&REQUEST=GetRecordById&outputSchema=http://www.w3.org/ns/dcat%23&ID=e4a406fa-9225-4e9b-a2ac-e42820093805 for a direct link to a single MD in csw-dcat format

File: .circleci/config.yml [hunk 1]
```diff
@@ -26,6 +26,7 @@ jobs:
           name: Install python dependencies
           # FIXME: For now, we fix setuptools due to https://github.com/etalab/data.gouv.fr/issues/1041
           command: |
+            python --version
```
Comment (quaxsze) [inline .circleci/config.yml:29]: Why do you need to print python version here?

**Review by quaxsze — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (ThibaudDauce) [reply .circleci/config.yml]: It was for debug (I had problems with Python 3.7 in CI/production since we are on 3.9 in local dev)

**Pull request merged**